### PR TITLE
add 1.4.1 version doc

### PR DESCRIPTION
**Pull request by tomsun28**
## Checklist

- [x]  I hereby agree to the terms of the [HertzBeat CLA](https://gist.github.com/tomsun28/511c04e7643901cb550bb6ecc75a661b)
- [ ]  I have read the [Contributing Guide](https://hertzbeat.com/docs/others/contributing/)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

### CODE REVIEW

File: home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-08-14-hertzbeat-v1.4.0.md [hunk 1]
```diff
@@ -94,10 +93,10 @@ HertzBeat 提供云边协同能力，可以在多个隔离网络部署边缘采
 3. 部署采集器集群
 
 ```
-docker run -d -e IDENTITY=custom-collector-name -e MANAGER_HOST=127.0.0.1 -e MANAGER_PORT=1158 --name hertzbeat-collector tancloud/hertzbeat-collector
+docker run -d -e IDENTITY=custom-collector-name -e MANAGER_HOST=127.0.0.1 -e MANAGER_IP=1158 --name hertzbeat-collector tancloud/hertzbeat-collector
 ```
 - `-e IDENTITY=custom-collector-name` : 配置此采集器的唯一性标识符名称，多个采集器名称不能相同，建议自定义英文名称。
-- `-e MANAGER_HOST=127.0.0.1` : 配置连接主HertaBeat服务的对外IP。
+- `-e MANAGER_IP=127.0.0.1` : 配置连接主HertaBeat服务的对外IP。
```
Comment (zqr10159) [inline home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-08-14-hertzbeat-v1.4.0.md:99]: 应为HertzBeat

File: home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+2. 浏览器访问 `http://localhost:1157` 即可开始，默认账号密码 `admin/hertzbeat`
+
+3. 部署采集器集群
+
+```
+docker run -d -e IDENTITY=custom-collector-name -e MANAGER_HOST=127.0.0.1 -e MANAGER_PORT=1158 --name hertzbeat-collector tancloud/hertzbeat-collector
+```
+- `-e IDENTITY=custom-collector-name` : 配置此采集器的唯一性标识符名称，多个采集器名称不能相同，建议自定义英文名称。
+- `-e MANAGER_HOST=127.0.0.1` : 配置连接主HertaBeat服务的对外IP。
```
Comment (zqr10159) [inline home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md:78]: 应为HertzBeat

File: home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+> 欢迎大家一起贡献你使用过程中自定义的通用监控类型监控模版。
+
+- Site Monitor, Port Availability, Http Api, Ping Connectivity, Jvm, SiteMap Full Site, Ssl Certificate, SpringBoot, FTP Server
+- Mysql, PostgreSQL, MariaDB, Redis, ElasticSearch, SqlServer, Oracle, MongoDB, Damon, OpenGauss, ClickHouse, IoTDB, Redis Cluster
+- Linux, Ubuntu, CentOS, Windows
+- Tomcat, Nacos, Zookeeper, RabbitMQ, Flink, Kafka, ShenYu, DynamicTp, Jetty, ActiveMQ
+- Kubernetes, Docker
+- Huawei Switch, HPE Switch, TP-LINK Switch, Cisco Switch
+- and more for your custom monitoring.
+- Notifications support `Discord` `Slack` `Telegram` `Mail` `Pinning` `WeChat` `FlyBook` `SMS` `Webhook`.
```
Comment (zqr10159) [inline home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md:158]: `Notifications support `Discord` `Slack` `Telegram` `Mail` `Dingtalk` `WeChat` `FlyBook` `SMS` `Webhook` `ServerChan`.`

File: home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+- 和更多自定义监控模版。
+- 通知支持 `Discord` `Slack` `Telegram` `邮件` `钉钉` `微信` `飞书` `短信` `Webhook`。
```
Comment (zqr10159) [inline home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-09-26-hertzbeat-v1.4.1.md:160]: 通知支持 `Discord` `Slack` `Telegram` `邮件` `钉钉` `微信` `飞书` `短信` `Webhook` `Server酱`。

File: home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-08-14-hertzbeat-v1.4.0.md (hunk 1 above)
Comment (tomsun28) [reply home/i18n/zh-cn/docusaurus-plugin-content-blog/2023-08-14-hertzbeat-v1.4.0.md:99]: 收到🫡

**Pull request merged**